src/main/java/frc/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -9,7 +10,8 @@ public class IntakeConstants {
     public static final double kP = 2;
     public static final double kI = 0;
     public static final double kD = 0.2;
-    public static final double kF = 0;
+    public static final double kF = 0; // not relevant anymore
+    public static final Slot0Configs PIDGains = new Slot0Configs().withKP(IntakeConstants.kP).withKI(IntakeConstants.kI).withKD(IntakeConstants.kD);
```

Comment:
You don't need to individual constants, just put the numbers on this line instead.